src/plinder/data/utils/annotations/get_ligand_validation.py

```diff
@@ -49,51 +54,73 @@ class SystemValidationThresholds(BaseModel):
     """
 
     entry_thresholds: EntryValidationThresholds = EntryValidationThresholds(
-        resolution=3.5, rfree=0.45, r=0.4, r_minus_rfree=0.05
+        max_resolution=3.5, max_rfree=0.45, max_r=0.4, max_r_minus_rfree=0.05
     )
     residue_thresholds: ResidueValidationThresholds = ResidueValidationThresholds(
-        rscc=0.8, rsr=0.3, average_occupancy=1.0
+        min_rscc=0.8, max_rsr=0.3, min_average_occupancy=1.0
     )
     residue_list_thresholds: dict[str, ResidueListValidationThresholds] = Field(
         default_factory=lambda: {
             "ligand": ResidueListValidationThresholds(
-                percent_rsr_under_threshold=100.0,
-                percent_rscc_over_threshold=100.0,
-                percent_occupancy_over_threshold=100.0,
-                check_if_has_alts=True,
+                max_num_unresolved_heavy_atoms=0,
+                max_alt_count=1,
+                min_average_occupancy=0.8,
+                min_average_rscc=0.8,
+                max_average_rsr=0.3,
+                max_percent_outliers_clashes=0.0,
+                min_percent_rsr_under_threshold=0.0,
+                min_percent_rscc_over_threshold=0.0,
+                min_percent_occupancy_over_threshold=0.0,
             ),
             "pocket": ResidueListValidationThresholds(
-                percent_rsr_under_threshold=90.0,
-                percent_rscc_over_threshold=90.0,
-                percent_occupancy_over_threshold=90.0,
-                check_if_has_alts=False,
+                max_num_unresolved_heavy_atoms=0,
+                max_alt_count=1,
+                min_average_occupancy=0.8,
+                min_average_rscc=0.8,
+                max_average_rsr=0.3,
+                max_percent_outliers_clashes=100.0,
+                min_percent_rsr_under_threshold=0.0,
+                min_percent_rscc_over_threshold=0.0,
+                min_percent_occupancy_over_threshold=0.0,
             ),
         }
     )
+    max_fraction_atoms_with_crystal_contacts: float = 0.0
```

Comment:
@OleinikovasV added fraction threshold here, should we try with strictest value (0) and see how many we lose? Does mean that if we change it we'll have to redo the system_pass_validation_criteria calculation on the index file - @tjduigna what's the best strategy here, we could move this to core instead of here to account for the fact that changing these thresholds does not actually need annotation generation rerun?

Reply:
I would change this to `max_atoms_with_crystal_contacts: int = 0` but this would be only for ligand.
Same would be for this `min_num_resolved_heavy_atoms: int = 2` - only for ligand criteria, I suppose.

Reply:
We should move these criteria to `splits` and change as necessary after annotation rerun as well

Reply:
We could do something like:
```python

entry = Entry.from_json(stored_entry_json)
entry.set_validation(val_file, cif_file, any_thresholds)
# pseudocode -> entry.to_json()
```
which avoids annotation generation itself by re-using what we already have, but allows us to pass new thresholds.

Operationally, this would additionally require updating the zipped entries and re-generating the index. Today there is no "update annotations" stage, but it would be relatively straightforward to write. `structure_qc` could be extended to only re-pack the `entries` zips and `make_plindex` is something that should have its own stage but currently runs inside of `make_mmp_index`.

Reply:
Could also be that we use the `pass_validation_criteria` columns only in the plindex and not in the entry jsons? Depends on what would mesh better with the dataloader

Reply:
Yeah we could opt not to save the `pass_validation_criteria` in the entry jsons and only evaluate them in `to_df` on index creation. Then we'd only be storing the data in one place, and we'd simply have to call `set_validation` in `structure_qc` prior to `Entry.to_df`. It would make sense to surface the validation configuration the same way we have it as for other configurable parts of the system.